init.lua

```diff
@@ -88,7 +88,7 @@ if core.get_modpath("default") and default then
 			"drawers_wood.png"),
 		groups = {choppy = 3, oddly_breakable_by_hand = 2},
 		sounds = drawers.WOOD_SOUNDS,
-		drawer_stack_max_factor = 4 * 8, -- normal chest size
+		drawer_stack_max_factor = 32, -- 4 * 8 normal chest size
```

Comment:
LuaJIT will evaluate this constant expression anyway, and even for Lua it's a single multiplication. This approach is much more error-prone due to potential calculation errors.
I'm not requesting a revert here but would like to point out that such sort of optimizations are not needed.

Reply:
thanks, I think it was in context of formspec that this approach was recommended.